### PR TITLE
Add logo widget

### PR DESCRIPTION
**Pull request by chakravarthi589**
@ajpmaclean : Please review

### CODE REVIEW

File: src/Java/Widgets/LogoWidget.java [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+
+  public static void main(String args[]) 
+  {
+  
+    vtkNamedColors colors = new vtkNamedColors(); 
+
+    //Circle Actor Color
+    double Circlecolor[] = new double[4]; 
+    colors.GetColor("Mint", Circlecolor);
+
```
Comment (ajpmaclean) [inline src/Java/Widgets/LogoWidget.java]: Use the same colours as the C++ example. This ensures consistency for users.
i.e 
``` java
double bkg[] = new double[]{0.2, 0.3, 0.4, 1.0};
int boxColor[] = new int[]{255, 127, 100, 255};
int circleColor[] = new int[4]; 
colors.GetColor("Black", circleColor);
```
So there you have your colors to use for `drawing`and `ren`. I haven't tested using `int[4]` but it should work. If not use `double[4]` and divide the ints in [255, 127, 100, 255] by 255.0.

**Comment by ajpmaclean** [comment]
Just add the background colour, I've had a look at the coffee for the VTK class, it only uses three doubles not an array fit the colours.

**Comment by chakravarthi589** [comment]
I have added the background color as suggested. Please review .

**Comment by ajpmaclean** [comment]
Thanks

**Pull request merged**